server.py

```diff
@@ -14,20 +14,14 @@
 
 # BattleshipRequestHandler is an extension of BaseHTTPRequestHandler
 class BattleshipRequestHandler(BaseHTTPRequestHandler):
-    def do_GET(self):
-        self.send_response(200)
```

Comment:
By removing these, are we actually sending any kind of response back to the client?

Reply:
I think it means that it'll 500 if it receives a GET. I'll add it back if we still need them, I wasn't sure for now